manifests/claudie/ansibler.yaml

```diff
@@ -18,8 +18,41 @@ spec:
         app.kubernetes.io/name: ansibler
     spec:
       terminationGracePeriodSeconds: 1320
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 3000
+        fsGroup: 2000
+      volumes:
+        - name: data
+          emptyDir: {}
+        - name: temp
+          emptyDir: {}        
       containers:
         - name: ansibler
+          volumeMounts:
+          - mountPath: /bin/services/ansibler/server/clusters
+            name: data
+          - mountPath: /.ansible
+            name: temp                  
+          securityContext:
+            capabilities:
+              drop:
+                - all
+              add:
+                - chown
```

Comment:
Were you able to identify the minimum amount of capabilities in order to run Claudie components?
How did you manage this?

Reply:
I followed this guide: https://cloud.redhat.com/blog/linux-capabilities-in-openshift

In short:
1. Exec into a pod and run `cat /proc/$PID/status | grep Cap`
2. Decode the hex in `capsh --decode`. This will output the cap used by the process
3. Apply them in the manifest and test :)

Reply:
So apparently I was wrong. I did a simple test and dropped all `CAP_`'s in each of the microservice - and I didn't saw any distruptions (tested localy). It must mean that the method I shared with you is describing something else. Like "what are the cap's that the current process haves" and not "what cap the current process is using".

Reply:
gotcha, thanks Adam
That's exactly what I was wondering. Now it's clear :D